test/util.spec.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+const ipldGit = require('../src')
+const multihash = require('multihashes')
+const CID = require('cids')
+
+describe('IPLD format util', () => {
+  const tagNode = {
+    gitType: 'tag',
+    object: {'/': new CID('z8mWaHQaEAKd5KMRNU3npB3saSZmhFh3e').buffer},
+    type: 'commit',
+    tag: 'v0.0.0',
+    tagger: {
+      name: 'John Doe',
+      email: 'johndoe@example.com',
+      date: '1497302532 +0200'
+    },
+    message: 'A message\n'
+  }
+
+  it('.serialize and .deserialize', (done) => {
+    ipldGit.util.serialize(tagNode, (err, serialized) => {
+      expect(err).to.not.exist()
+      expect(Buffer.isBuffer(serialized)).to.equal(true)
```

Comment:
Nit-pick: This can be written as `expect(Buffer.isBuffer(serialized)).to.be.true()`. There's no need to change it, it's just a note for future cases.